.ci-builds/.azure-pipelines.yml

```diff
@@ -57,7 +57,7 @@ jobs:
     - script: |
         cd icu4c/source && \
         ICU_DATA_FILTER_FILE=../../.ci-builds/data-filter.json ./runConfigureICU Linux && \
-        make -j -l2.5 tests && \
+        make -j2 tests && \
```

Comment:
Please add a comment like
```
# Revert this check from PR #2456 `make -j -l2.5` to `make -j2` because with more parallelism
# this check became flaky.
# The build apparently was not done copying one or another .ucm file before calling makeconv for it,
# although the makefile has appropriate dependencies.
```